frontend/packages/webterminal-plugin/src/components/cloud-shell/__tests__/CloudShellExec.spec.tsx

```diff
@@ -27,8 +27,12 @@ const cloudShellExecProps: CloudShellExecProps = {
 };
 
 describe('CloudShellExec', () => {
+  beforeAll(() => {
+    configure({ testIdAttribute: 'data-test' });
```

Comment:
There is no need to add `configure` in `beforeAll`, this can be added after imports. Can you address it for all the specs files.